packages/esm-patient-programs-app/src/programs/programs-actions-menu.component.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+import { showModal, useLayoutType } from '@openmrs/esm-framework';
+import { Layer, OverflowMenu, OverflowMenuItem } from '@carbon/react';
+import React, { useCallback } from 'react';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+
+interface ProgramActionsProps {
+  patientUuid: string;
+  programEnrollmentId: string;
+}
+
+export const ProgramsActionsMenu = ({ patientUuid, programEnrollmentId }: ProgramActionsProps) => {
+  const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+
+  const launchEditProgramsForm = useCallback(
+    () => launchPatientWorkspace('programs-form-workspace', { programEnrollmentId }),
+    [programEnrollmentId],
+  );
+
+  const launchDeleteProgramDialog = () => {
+    const dispose = showModal('program-delete-confirmation-modal', {
+      closeDeleteModal: () => dispose(),
+      programEnrollmentId,
+      patientUuid,
+    });
+  };
+
+  return (
+    <Layer>
+      <OverflowMenu
+        name={t('editOrDeleteProgram', 'Edit or delete program')}
+        aria-label={t('editOrDeleteProgram', 'Edit or delete program')}
+        size={isTablet ? 'lg' : 'sm'}
+        flipped
+      >
```

Comment:
```suggestion      
      <OverflowMenu
        align="left"
        aria-label={t('editOrDeleteProgram', 'Edit or delete program')}
        flipped
        size={isTablet ? 'lg' : 'sm'}
      >
```

The `OverflowMenu` should be left-aligned so it's content does not flow over the width of the datatable. Additionally, we should remove the `name` prop as it's not a known prop of the [OverflowMenu component](https://react.carbondesignsystem.com/?path=/docs/components-overflowmenu--overview).